rest-service/manager_rest/resource_manager.py

```diff
@@ -266,29 +266,20 @@ def _get_queued_executions(self, finished_execution):
 
         if finished_execution.deployment:
             deployment_id = finished_execution.deployment.id
-            same_dep_executions = self.sm.list(
-                models.Execution,
-                filters={
-                    'status': ExecutionState.QUEUED_STATE,
-                    'deployment_id': deployment_id,
-                },
-                sort=sort_by,
-                get_all_results=True,
-                all_tenants=True,
-                locking=True,
-            ).items
-            other_queued = self.sm.list(
+            queued_executions = self.sm.list(
                 models.Execution,
                 filters={
                     'status': ExecutionState.QUEUED_STATE,
-                    'deployment_id': lambda col: col != deployment_id,
+                    'is_system_workflow': False
                 },
-                sort=sort_by,
+                sort=OrderedDict([
```

Comment:
needs to be ordered, so that executions are sorted by deployment first, and only within those groups by the timestamp